src/callbacks/onPreprocess.js

```diff
@@ -26,8 +26,5 @@ export default function onPreprocess() {
 
     //Insert groupings into data to draw empty rows in which to draw groupings.
     if (this.config.y.grouping) defineGroupingData.call(this);
-    else {
-        delete this.groupings;
-        this.config.range_band = this.initialSettings.range_band;
```

Comment:
Don't need the range_band reset anymore?

Reply:
No, just needed to move the y-domain definition to onPreprocess rather than onDraw.  Or something.  Definitely addressed in #114.